latex/content/debug.tex

```diff
@@ -1,6 +1,6 @@
 \subsection{Debug}
 
-\begin{frame}[fragile]{\lstinline+\\overfullrule+}
+\begin{frame}[fragile]{\texttt{\textbackslash overfullrule}}
```

Comment:
I can remember fixing the font color for some kind of titles, I think it was the box titles.

Ah yes, there it is: https://github.com/pep-dortmund/toolbox-workshop/blob/master/common/settings.tex#L16

You should be able to use beamer styles to do the same for frame titles.